Uruguay/URY_extract_microdata_excel_to_csv.py

```diff
@@ -0,0 +1,41 @@
+# Databricks notebook source
+# MAGIC %run ../utils
+
+# COMMAND ----------
+
+from tqdm import tqdm
+import openpyxl
+import pandas as pd
+
+COUNTRY = 'Uruguay'
+microdata_csv_dir = prepare_microdata_csv_dir(COUNTRY)
+filename = input_excel_filename(COUNTRY)
+
+# TODO: Figure out the difference between cen and Municipalidades
+disaggregated_data_sheets = ['Datos']
+
+for sheet in tqdm(disaggregated_data_sheets):
+    csv_file_path = f'{microdata_csv_dir}/{sheet}.csv'
+    df = pd.read_excel(filename, sheet_name=sheet, header=0)
+
+    # Handle unnamed or null named columns
+    header = [col_name for col_name in df.columns if is_named_column(col_name)]
+    df = df[header]
+    df.columns = [col.strip() for col in header]
+        
+    # Normalize cells
+    df = df.applymap(normalize_cell)
+
+    # Remove rows where all values are null
+    df = df.dropna(how='all')
+    
+    # Write to CSV
```

Comment:
No need to comment on every single line of the code. As a general rule of thumb, use comment to explain the "why" as others can understand the "what" from reading the code.